.eslintrc

```diff
@@ -9,6 +9,8 @@
           "exports": "always-multiline",
           "functions": "always-multiline"
         }],
+        "function-paren-newline": ["error", "consistent"],
```

Comment:
The latest eslint configuration does not like some of my conventions around newlines. I think requiring consistently around parenthesis and curly brace newlines is important, so I kept that level of validation.